modules/osdk-cli-reference-olm-install.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-cli-reference.adoc
+
+[id="osdk-cli-reference-olm-install_{context}"]
+= install
+
+`olm install` subcommand installs OLM in your cluster.
+
+.`install` arguments
+[options="header",cols="1,3"]
+|===
+|Argument |Description
+
+|`--olm-namespace` string
+|The namespace where OLM is installed. Default: `olm`.
+
+|`--timeout <duration>`
+|The time to wait for the command to complete before it fails. Default: `2m0s`.
+
+|`--version` string
+|The version of OLM resources to be installed. Default: `latest`.
+
+|`-h, --help`
+|Usage help output
```

Comment:
nit: Please add ending punctuation.  I think the next three files have the same `--help` argument and the same one-liner description.